Formula/r/rsgain.rb

```diff
@@ -0,0 +1,29 @@
+class Rsgain < Formula
+  desc "ReplayGain 2.0 tagging utility"
+  homepage "https://github.com/complexlogic/rsgain"
+  url "https://github.com/complexlogic/rsgain/archive/refs/tags/v3.5.2.tar.gz"
+  sha256 "c76ba0dfaafcaa3ceb71a3e5a1de128200d92f7895d8c2ad45360adefe5a103b"
+  license "BSD-2-Clause"
+  head "https://github.com/complexlogic/rsgain.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ffmpeg"
+  depends_on "fmt"
+  depends_on "inih"
+  depends_on "libebur128"
+  depends_on "taglib"
```

Comment:
does it also support taglib 2?

Reply:
Not sure, but considering we cannot get taglib 2 shipped I'm not super worried about it.

Reply:
yeah, totally.